mapper/mapper_test.go

```diff
@@ -229,7 +232,7 @@ func TestReleaseCalendarMapper(t *testing.T) {
 		}
 
 		Convey("CreateReleaseCalendar maps correctly to a model Calendar object", func() {
-			calendar := CreateReleaseCalendar(basePage, params, releaseResponse)
+			calendar := CreateReleaseCalendar(basePage, params, releaseResponse, config.Config{DefaultMaximumSearchResults: 1000})
```

Comment:
Maybe we could define this `config` in the outer convey (setup/given) for easier maintenance?